cmd/admin-user-sa-generate.go

```diff
@@ -0,0 +1,120 @@
+/*
+ * MinIO Client (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"io/ioutil"
+
+	"github.com/minio/cli"
+	json "github.com/minio/mc/pkg/colorjson"
+	"github.com/minio/mc/pkg/probe"
+	"github.com/minio/minio/pkg/console"
+)
+
+var adminUserSAGenerateCmd = cli.Command{
+	Name:   "generate",
+	Usage:  "generate a new service account",
+	Action: mainAdminUserSAGenerate,
+	Before: setGlobalsFromContext,
+	Flags:  globalFlags,
+	CustomHelpTemplate: `NAME:
+  {{.HelpName}} - {{.Usage}}
+
+USAGE:
+  {{.HelpName}} TARGET PARENT-USER [POLICY_FILE]
+
+PARENT-USER:
+  The parent user.
+
+POLICY_FILE:
+  The path of the policy to apply for the new service account.
+  When unspecified, the policy of the parent user will be evaluated
+  instead for all type of service accounts requests.
+
+FLAGS:
+  {{range .VisibleFlags}}{{.}}
+  {{end}}
+
+EXAMPLES:
+  1. Add a new service account under the name of 'foobar' to MinIO server.
+     {{.Prompt}} {{.HelpName}} myminio foobar /tmp/policy.json
```

Comment:
provide an example without the policy

Reply:
> provide an example without the policy

done, https://github.com/minio/mc/pull/3105/commits/a5f3b1d6fcfbb938c1879f9c2465f7b9a8d7e87f